fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1127,9 +1213,23 @@ ACTOR Future<Void> postResolution(CommitBatchContext* self) {
 
 	self->toCommit.saveTags(self->writtenTags);
 
-	// Obtain previous committed versions for each affected tlog from sequencer
-	if (SERVER_KNOBS->ENABLE_VERSION_VECTOR) {
-		wait(getTPCV(self));
+	if (self->writtenTags.size() && !self->metadataMutationFromProxy &&
+	    SERVER_KNOBS->ENABLE_VERSION_VECTOR_TLOG_UNICAST) {
+		// confirm all serialized tags are sent to a tLog for which the previous commit version was obtained.
+		std::set<uint16_t> postResolutionTLogs;
+		self->toCommit.getLocations(self->writtenTags, postResolutionTLogs);
+		for (auto& t : postResolutionTLogs) {
+			if (self->writtenTLogs.find(t) == self->writtenTLogs.end()) {
+				TraceEvent("Tag in message does not have PCV!")
```

Comment:
It's good idea to use `TraceEvent(SevErrir, "TagHasNoPCV", pProxyCommitData->dbgid)`.
`SevError` to signify the error.
`dbgid` to assign proxy's id to the traceevent.

Reply:
ok